test-cli.py

```diff
@@ -472,10 +474,10 @@ def test(ctx):
     ctx.collect_worksheet(wuuid)
     # Add tags
     run_command([cl, 'wedit', wname, '--tags', 'foo', 'bar', 'baz'])
-    check_contains(['Tags: \\[\'foo\', \'bar\', \'baz\'\\]'], run_command([cl, 'ls', '-w', wuuid]))
+    check_contains(['Tags: \\[u\'foo\', u\'bar\', u\'baz\'\\]'], run_command([cl, 'ls', '-w', wuuid]))
```

Comment:
When I run (both locally on Ubuntu 14.04 w/Python 2.7.5) and on Circle it says that it needs the u's. Not sure why it would output unicode strings now and not before